### PR TITLE
Convert CCWF examples to use Job API

### PR DESCRIPTION
**Pull request by SYangster**
### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/hello-world/step-by-step/cifar10/swarm/swarm.ipynb [hunk 1]
```diff
@@ -41,90 +41,90 @@
...
+    "from nvflare.apis.dxo import DataKind\n",
+    "from nvflare.app_common.aggregators.intime_accumulate_model_aggregator import InTimeAccumulateWeightedAggregator\n",
+    "from nvflare.app_common.ccwf.ccwf_job import CCWFJob, CrossSiteEvalConfig, SwarmClientConfig, SwarmServerConfig\n",
+    "from nvflare.app_common.ccwf.comps.simple_model_shareable_generator import SimpleModelShareableGenerator\n",
+    "from nvflare.app_opt.pt.file_model_persistor import PTFileModelPersistor\n",
+    "from nvflare.job_config.script_runner import ScriptRunner\n",
+    "\n",
+    "n_clients = 2\n",
+    "num_rounds = 3\n",
+    "train_script = \"../code/fl/train.py\"\n",
```
Comment (YuanTingHsieh) [inline examples/hello-world/step-by-step/cifar10/swarm/swarm.ipynb:111]: this will not work anymore as you pointed out

**Review by YuanTingHsieh — COMMENTED** [review]
Mostly good with one thing need to be fixed.

Another thing we could think of is to consolidate this with advanced/job_api
And if this ccwf is really hello-world level example

File: examples/hello-world/hello-ccwf/swarm_cse_script_runner_pt.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+from nvflare.app_common.aggregators.intime_accumulate_model_aggregator import InTimeAccumulateWeightedAggregator
+from nvflare.app_common.ccwf.ccwf_job import CCWFJob, CrossSiteEvalConfig, SwarmClientConfig, SwarmServerConfig
+from nvflare.app_common.ccwf.comps.simple_model_shareable_generator import SimpleModelShareableGenerator
+from nvflare.app_opt.pt.file_model_persistor import PTFileModelPersistor
+from nvflare.job_config.script_runner import ScriptRunner
+
+if __name__ == "__main__":
+    n_clients = 2
+    num_rounds = 3
+    train_script = "src/hello-ccwf_cifar10_fl.py"
```
[the remark this answers is not in the document]
Comment (SYangster) [reply examples/hello-world/hello-ccwf/swarm_cse_script_runner_pt.py:27]: I think keeping in src is good for consistency with all the other hello-examples

File: examples/hello-world/step-by-step/cifar10/swarm/swarm.ipynb (hunk 1 above)
Comment (SYangster) [reply examples/hello-world/step-by-step/cifar10/swarm/swarm.ipynb]: thanks, fixed to copy files over

File: examples/hello-world/hello-ccwf/swarm_script_runner_np.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        server_config=SwarmServerConfig(num_rounds=num_rounds),
+        client_config=SwarmClientConfig(
+            executor=ScriptRunner(script=train_script, framework=FrameworkType.NUMPY),
+            aggregator=aggregator,
+            persistor=NPFileModelPersistor(),
+            shareable_generator=SimpleModelShareableGenerator(),
+        ),
+    )
+
+    # job.export_job("/tmp/nvflare/jobs/job_config")
```
Comment (YuanTingHsieh) [inline examples/hello-world/hello-ccwf/swarm_script_runner_np.py:39]: minor: can uncomment this to be consistent with text.

**Review by YuanTingHsieh — DISMISSED** [review]
one minor thing , otherwise LGTM

File: examples/hello-world/hello-ccwf/swarm_script_runner_np.py (hunk 3 above)
Comment (SYangster) [reply examples/hello-world/hello-ccwf/swarm_script_runner_np.py]: fixed

**Comment by SYangster** [comment]
I need to fix CI for hello-ccwf

**Comment by SYangster** [comment]
The advanced/job_api already has pt versions of swarm and ccwf cyclic. This hello-ccwf is simpler as it uses numpy.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**